starter-code/index.css

```diff
@@ -0,0 +1,155 @@
+
+  /* ---- typography ---- */
+
+
+  
+  /* ---- layout ---- */
+  
+  .row {
```

Comment:
Are you overwriting Bootstratp's classes? It's not a good idea and gives you some weird performance. If you want to use your own classes, select diferent names that the ones from Bootstrap. That said, if you are going to use Bootstrap's grid, then use its classes for grid.